package.json

```diff
@@ -23,6 +23,8 @@
     "clean": "rm -rf umd es"
   },
   "dependencies": {
+    "@rollup-umd/build": "^1.1.1",
+    "canvas": "^2.6.1",
```

Comment:
This lib only used in CLI shouldn't be listed here. CI will install it all the time.

Reply:
But if a user download the repo and want to run it locally, only the commands `npm install` and `npm run build` will fail without canvas lib in dependencies.

Reply:
see https://docs.npmjs.com/files/package.json#optionaldependencies

Reply:
Already moved to devDeps in the `master` branch.